packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -826,12 +750,6 @@ class HotRunner extends ResidentRunner {
           printError('Reassembling ${view.uiIsolate.name} failed: $error');
           return;
         }
-        try {
-          await view.uiIsolate.uiWindowScheduleFrame();
```

Comment:
What is the rationale for this being removed? If it's not related to the way how we monitor changes, perhaps it should land separately?

Reply:
Filed https://github.com/flutter/flutter/issues/29182 to track.

Reply:
Added back for now